### DISCUSSION TITLE
Eigene Platine für das Daikin-Rotex-HPSU-CAN Projekt

### DISCUSSION THREAD

**Discussion by dornieden** [Trunks1982/Daikin-Rotex-HPSU-CAN#35] [Show and tell]
Hallo zusammen,

ich freue mich total über dieses Projekt und hoffe auch einen kleinen Teil beitragen zu können.
In den letzten Tagen habe ich eine Platine layoutet die ich aktuell mit meiner Wärmepumpe teste.

Die Idee ist, dass nur noch ein Anschlusskabel von der Wärmepumpe zur Platine angeschlossen werden muss und damit:
* Stromversorgung 24V DC
* CAN Kommunikation und
* Ansteuerung von Heiz- und Kühlkreislauf über zwei Relais möglich ist.

Der ESP32 muss nur noch auf die Platine aufgelötet werden oder mit Hilfe von Steckleisten gesteckt werden.

Bei Interesse stelle ich hier gerne weitere Details zur Verfügung.

Wenn alle Tests erfolgreich sind, dann würde ich zumindest für @Trunks1982  auch eine Platine kostenlos zur Verfügung stellen wenn er mag, als kleinen Dank für seine Arbeit hier.

Ich freue mich über euer Feedback!
Bernd

<img width="862" alt="Bildschirmfoto 2024-04-19 um 12 53 51" src="https://github.com/Trunks1982/Daikin-Rotex-HPSU-CAN/assets/47507239/ba77c12e-dd1f-48b5-a740-1d44ce1a4a4d">

**Comment by Trunks1982** [comment]
Sieht mega aus.Wo holst du dann die 24v her?

**Reply by Trunks1982** [reply]
Frage wäre es nicht möglich einen Lan Anschluss zu integrieren wo mittels poe strom zugeführt wird und daten per lan kommen?

**Comment by dornieden** [comment]
> Siegt mega aus.Wo holst du dann die 24v her?

Vielen Dank!
Die 24V sollten sich ebenfalls vom J13 Stecker der Rotex abgreifen lassen: "CAN-VCC"

**Reply by Nitrox912** [reply]
Die Idee ist toll. Aber wie sieht es mit einem LAN-Anschluss aus? Wäre das grundsätzlich machbar? Und könnte man die Platine auch so konstruieren, dass man einen ESP32 einfach aufsteckt?

**Reply by dornieden** [reply]
Die Platine im Bild zeigt zwar eine Stiftleiste, verlötet habe ich aber eine Buchsenleiste. Auf diese kann der ESP32 ganz einfach aufgesteckt werden und auch jederzeit wieder entfernt werden. Das funktioniert soweit richtig gut.
Was leider nicht gut klappt ist die Idee mit den 24V. Der Spannungswandler auf der Platine wird zu heiß und der ESP32 mag damit aus irgendeinem Grunde auch nicht richtig aufstarten.

Ich verwende die Platine also weiterhin erst mal mit einer USB-C Verbindung zum ESP32.

Ein LAN Port wäre auch möglich. Entweder mit einem dedizierten ESP32 der bereits einen POE Port hat (die sind aber ziemlich teuer und passen nicht mehr zu der Buchsenleiste).
Oder aber mit einem neuen Platinenlayout. Da ich mit der WiFi Verbindung aber erst mal recht zufrieden bin bleibt es erst mal bei diesem Design.

Falls jemand eine Platine haben möchte, dann schicke ich euch ein Exemplar zum Selbstkostenpreis gerne zu. Ich habe noch 3 Stück bei mir.

Der nächste Schritt ist jetzt das Layout eines passenden Gehäuses zur Aufnahme der Platine.

**Reply by magic2910** [reply]
Gibt es eine überarbeitete Platine ohne die 24v? Bzw wird es die geben?

**Reply by dornieden** [reply]
Wenn Interesse besteht, dann mache ich gerne eine Version ohne 24V.

Der Relais- und CAN-Part der Platine funktionieren soweit übrigens einwandfrei.

**Reply by magic2910** [reply]
Ah ok, das heisst wenn ich den 24V nicht nutze, habe ich keine Probleme?
Hättest du noch eine Platine übrig?

**Reply by dornieden** [reply]
genau, ohne Anschluss der 24V funktioniert die aktuelle Version ebenfalls problemlos.
Ja, hab noch 3 Platinen übrig. Schreib mir am besten ne kurze Mail an info@smartpassivehouse.com

**Reply by Tom88s** [reply]
Ich finde deine Platine auch sehr interessant :) 
Versteh noch nicht ganz den Sinn der Relais....
Würde die Platine aber gerne ohne den 24V Teil für meine HA Integration nutzen.
Hast du auch noch eine für mich oder sind schon alle vergeben?

Grüße Thomas

**Reply by dornieden** [reply]
2 Platinen habe ich noch. Schreib mir gerne eine kurze Mail an info@smartpassivehouse.com.

Bitte beachte aber, dass Du die 24V nicht anschließen, sondern statt dessen den ESP32 über den USB Anschluss versorgen musst.

Viele Grüße
Bernd

**Comment by magic2910** [comment]
Welchen ESP32 verwendest du auf dem Board? Ich bekomme den ESP32 S3 DevKitC 1 N16R8 ESP32 S3 WROOM1 N16R8 
Nicht zum laufen, er startet nicht, wenn er auf dem Board gesteckt ist.

**Reply by magic2910** [reply]
Ok, mein Fehler, hatte den esp verkehrt, man sollte ihn am besten auf die Unterseite löten, ansonsten ist der USB-C schwer zugänglich.

**Reply by dornieden** [reply]
Freut mich das es jetzt bei Dir läuft. 
Mit der Buchsenleiste die ich Dir beigelegt habe klappt der USB Anschluss über den Schraubanschlüssen bei mir ganz knapp. In einer nächsten Version würde ich aber den 24V Teil weglassen und dann eine Lücke bei den Schraubanschlüssen vorsehen wo das USB Kabel mehr Platz hat.

**Reply by magic2910** [reply]
Das klingt gut! Gib Bescheid, sobald die neue Platine erhältlich ist :) 
Vielen Dank!

**Reply by Tom88s** [reply]
Wie herum gehört der ESP auf die Platine?
Damit ich nichts falsch mach sobald meine Platine ankommt 😉
Freu mich schon aufs basteln 😜 damit ich endlich Fhem ablösen kann

**Reply by magic2910** [reply]
USB C auf die Seite der Anschlüsse, dass man quasi nur ganz knapp den Stecker reinbekommt

**Reply by Tom88s** [reply]
Dann besorg ich mir gleich ein gewinkeltes Kabel. Danke für die Info

Funktionieren die Relais auch mit der USB C Spannungsversorgung oder ist der Teil der Platine einfach nicht nutzbar?

**Reply by magic2910** [reply]
Winkelstecker ist eine gute Idee! Danke!

**Comment by Trunks1982** [comment]
Hi waren auf der Platine eigentlich schon relais verbaut oder nur die gpio anschlüsse?

**Reply by Tom88s** [reply]
Ja die Relais sind schon drauf

**Reply by Trunks1982** [reply]
Das hört sich gut an.Fehlen dann nur noch für sg zwei relais.

**Reply by Tom88s** [reply]
Die SG Relais sind bei mir im Stromspeicher

**Comment by Trunks1982** [comment]
@dornieden hast du schon eine neue Platine entwickelt?

**Reply by dornieden** [reply]
Gutes Timing - gerade heute sind die Platinen der neuen Version bei mir eingetroffen. Werde sie jetzt fertig löten (Relais waren diesmal nicht lieferbar, daher löte ich sie von der alten Version um) und dann intensiv testen. 

Diesmal ist kein Spannungswandler verbaut, dafür aber eine bessere Anordnung der Anschlüsse. 

Ein passendes Gehäuse für den 3D Drucker habe ich auch bereits fertig und kann dann beides hier bald zur Verfügung stellen. 

![image](https://github.com/Trunks1982/Daikin-Rotex-HPSU-CAN/assets/47507239/519dfe11-5de6-4dfe-9f1b-5886945ffad5)

**Reply by Trunks1982** [reply]
Mal ne frage ist es möglich nen esp32 direkt auf die Platine zu machen?

**Reply by dornieden** [reply]
Du kannst den ESP einerseits natürlich direkt auf die Platine löten. Ich nutze die Headerboards einfach dazu den ESP leichter austauschen zu können. 

Alternativ könnte man den ESP32 auch direkt als SMD Bauteil auf der Platine verbauen lassen. 

Dann ist aber nicht mehr unbedingt die PIN Gleichheit für die hier geschriebene SW gegeben. 

Und so Dinge wie Flashen gehen dann auch nicht unbedingt einfach. Du bräuchtest dann auch wieder zusätzliche Bauteile auf der Platine dafür. 

Diese ESP32 Entwicklerboards sind schon ganz nett bezüglich Preis und Features.

**Reply by Trunks1982** [reply]
Ok.Die Pin Belegung wäre ja nicht das schlimmste.Nur flashen sollte schon wie mit einem normalen esp funktionieren.

**Reply by Trunks1982** [reply]
Komm doch mal in die Telegramm Gruppe dann könnte man ja mal wünsche usw diskutieren.Sind schon nen paar drin.😊

**Reply by oliverlinsenmaier** [reply]
@dornieden Ich hätte Interesse an dieser Platine, damit ich meine Heizung Smart machen. Wie kann ich teilhaben?

**Comment by dornieden** [comment]
Hallo zusammen,

seit einigen Tagen habe ich eine neue Version der Platine (v0.3) im Einsatz die ich euch hier gerne vorstellen möchte:

Geändert hat sich im Vergleich zur ersten Version:

- Spannungsversorgung für den ESP32 entfernt. Dieser wird jetzt wieder per USB-C versorgt
- Bessere Anordnung der Komponenten um Anschlüsse zu vereinfachen
- Platinenbeschriftung um richtige Anordnung zu vereinfachen

Neben der Platine selbst habe ich auch ein einfaches Gehäuse erstellt. Mit diesem bin ich allerdings noch nicht wirklich glücklich. Für mich selber werde ich daher wohl eher dazu übergenen die Platine auf einem Platinenträger im Hutschinengehäuse einer Unterverteilung unterzubringen.

Ihr findet alle notwendigen files zum Bestellen eurer eigenen Platine hier angehängt. Ebenso wie die Gehäuse-Dateien für den 3-D Druck.

Ich freue mich über eure Kommentare wenn ihr mit der neuen Version der Platine etwas anfangen könnt!

Dateien Platine:
[PickAndPlace_PCB_HPSU-CAN-ESPHOME_v03_2024-08-12.csv](https://github.com/user-attachments/files/16580819/PickAndPlace_PCB_HPSU-CAN-ESPHOME_v03_2024-08-12.csv)
[Gerber_HPSU-CAN-ESPHOME_PCB_HPSU-CAN-ESPHOME_v03_2024-08-12.zip](https://github.com/user-attachments/files/16580820/Gerber_HPSU-CAN-ESPHOME_PCB_HPSU-CAN-ESPHOME_v03_2024-08-12.zip)
[BOM_HPSU-CAN-ESPHOME_2024-08-12.csv](https://github.com/user-attachments/files/16580821/BOM_HPSU-CAN-ESPHOME_2024-08-12.csv)
[HPSU-CAN-ESPHOME.pdf](https://github.com/user-attachments/files/16580822/HPSU-CAN-ESPHOME.pdf)
<img width="554" alt="Bildschirmfoto 2024-08-12 um 13 05 28" src="https://github.com/user-attachments/assets/fda9cc55-63de-4862-963a-ea39f58e5028">
![PCB_PCB_HPSU-CAN-ESPHOME_v03_2024-08-12](https://github.com/user-attachments/assets/2076ea76-326e-4f0e-99aa-a5ababa0d0bd)

Gehäuse:
[Case.zip](https://github.com/user-attachments/files/16580849/Case.zip)

**Comment by cicatrix87** [comment]
Da das von Bernd nicht komplett ist, habe ich ein eigenes entworfen.

[revised_case.zip](https://github.com/user-attachments/files/17011163/revised_case.zip)

**Comment by OldButTired** [comment]
SUPER!!!! Echt stark was Ihr da geschaffen habt!
Leider bin ich hundsalt aber müde und habe kaum Zeit zum basteln! Kann ich Euch die Platine mit ESP usw.  fertig abkaufen? Kauf wie auf dem Flohmarkt ohne Garantie? Das wäre echt stark, meine Rotex HPSU Compact von 2013 wird langsam älter und gerade heute habe ich die brühmten "zwei E8100-Kondensatoren" getauscht weil die Temperaturwerte ständig gesprungen sind und die Anlage warscheinlich schon lange auf Heizstab gelaufen ist. Ich habe es erst beim Umgraben im Garten gemerkt, weil der Verdichter immer mal an und gleich wieder ausging. Hätte ich die Anlagenwerte im Homeassistant mitgeschrieben, hätte ich gleich gemerkt das da was nicht stimmt! Vielen Dank!

**Reply by Cicatr1x** [reply]
Am einfachsten holst du dir ein M5 Stack Bundle aus Atom S3 und dem passenden CAN-Modul.

[M5 Stack AtomS3](https://botland.de/atom-serie/22676-atoms3-lite-dev-kit-esp32s3-entwicklungskit-m5stack-c124-6972934174055.html)
[M5Stack A103 CAN Modul](https://botland.de/atom-serie/24381-atomic-can-bus-base-ca-is3050g-can-erweiterungsmodul-fur-m5atom-m5stack-a103-6972934175014.html)

**Reply by OldButTired** [reply]
Vielen Dank!
Aber  Dornieden hat sich doch letztes Jahr so viel Mühe mit dem Board und den zwei Relais gegeben, alleine schon um den Aufwand zu würdigen würde ich am liebsten dieses bei Ihm bestellen.
Ausserdem: Meine Bastelzeit ist mit 3Kindern usw, ultrakurz und die Variante von Dornieden funktioniert augenscheinlich, keine andere Hardware, kein Bibliothekensuchspiel... ;)

Aber vielen Dank für Die schnelle Info!

**Reply by Cicatr1x** [reply]
Tag ihn oder frag per Telegram nach, sonst hast du keine Chance Bernd zu erreichen. Ob er noch Platinen hat ist die andere Frage, glaube ich hab eine der letzten bekommen.

Bezüglich Bastelzeit ist das einerlei. Die M5 Stack Kombination ist genauso einfach und du musst dich sowohl bei der Platine als auch dort mit ESPhome auseinander setzen.

Der Software Part ist also identisch, zu Bernds Platine wird ebenfalls ein esp32 s3 benötigt.

**Reply by Trunks1982** [reply]
Einen esp 33 s3 brauchste bei dem Board ja schon.

**Comment by NetRat88** [comment]
Hallo Zusammen, ich bin gerade auf dieses tolle Projekt gestoßen, da ich in unserem Mietshaus ohne Eingriff auf die Platine der Altherma 3 (EHS(X/H)(B)-D) nach einem einfachen Weg gesucht habe diese in Home Assistant zu integrieren.

Ich dachte erst, Netzwerkkabel rein und ab ins Netzwerk und dann über die Ocecta Daikin App integrieren, aber das scheint wohl kein Netzwerkanschluss zu sein, sondern der J13 Anschluss?

Dabei bin ich auf eine RJ45 Buchse hinter der Abdeckung gestoßen und glaube hier den CAN Bus gefunden zu haben? Gibt es denn mittlerweile eine Plug and Play Lösung, die man kaufen könnte, wo man einfach ein RJ45 Kabel einsteckt und diese auch durch die Buchse mit Strom versorgt wird? Ich wäre extrem dankbar für eure Unterstützung die Luft/Wärmepumpe bzw. Fußbodenheizung ins HAOS zu bekommen![image](https://github.com/user-attachments/assets/a823d769-52b9-479f-9255-824bcdbf7912)

**Comment by KalaNagHTD** [comment]
Auf diesem RJ45 liegt leider kein CAN-Bus.

**Reply by NetRat88** [reply]
Sicher? Laut Anschlussplan in der Anleitung scheint es auf das J13 Board zu gehen oder?![image](https://github.com/user-attachments/assets/13b304ef-2771-44f6-8ee1-deba4be195cf)

**Reply by KalaNagHTD** [reply]
Mmh…auf der Platine BM2C sitzt ein CAN-Transceiver, dessen Eingangssignal vom zentralen Mikroprozessor kommt, und dessen CAN-Ausgang nur zum J13 geht. Mein Verständnis ist, dass die Signale am RJ45 vor dem Transceiver abgegriffen werden.

**Reply by NetRat88** [reply]
Also ich hab noch einen M5 Stack Lite rumliegen und mir mal den M5Stack ATOMIC CAN Base, CA-IS3050G bestellt. Werde daran mal ein abgeschnittenes RJ45 Kabel anschließen und mal schauen was passiert. Kaputt machen sollte man über den CAN Anschluss ja nichts machen können!? 🙈
![image](https://github.com/user-attachments/assets/39b66854-a768-4277-9b77-1c9c94a1f6ca)

**Comment by KalaNagHTD** [comment]
Offenbar muss ich mich revidieren. Auch das Bedienteil ist mit „CAN“ gekennzeichnet, wobei „4 wire“ für CAN high, CAN low, Versorgungsspannung und Masse stehen könnte. Von den 8 Adern des Netzwerkkabels (rechts unten im Bild) wären dann vier ohne Funktion.

Mit einem Multimeter lässt sich oft schon viel herausfinden:
* Wenn zwischen zwei Leitungen im ausgeschalteten Zustand ca. 60 Ω liegenn, sind das sehr wahrscheinlich CAN_H/CAN_L (zwei 120-Ω-Abschlüsse parallel).
* Zwischen Versorgung und Masse misst du im Betrieb etwa 19–30 V.
* Gegen Masse liegen CAN_H und CAN_L im Ruhezustand typischerweise beide bei etwa 2,5 V.

<img width="3024" height="4032" alt="IMG_3758" src="https://github.com/user-attachments/assets/51d8b39b-dc68-4592-aab6-a784be048c60" />

**Reply by NetRat88** [reply]
Mega! Vielen Dank für die Unterstützung und Hilfe! Damit sollte ich doch arbeiten können.
Ich werde Rückmeldung geben.

Du sagtest Versorgungsspannung. Könnte ich dann sogar den Atom Lite direkt mit Strom versorgen, ohne ein zusätzliches USB-C Netzteil oder wäre das nicht zu empfehlen?

**Comment by NetRat88** [comment]
@KalaNagHTD Danke noch mal für deine Tipps bzgl. Multimetermessungen.
Ich habe inzwischen ein Patchkabel aufgeschnitten und die RJ45-Buchse meiner Altherma 3 **EHS(X/H)(B)-D / RoCon BM2C** mit einem Fluke 87V durchgemessen. Da ich zur Miete wohne, möchte ich nicht direkt an J13 oder der Platine arbeiten.

Das verwendete Patchkabel ist T568B belegt:

* Pin 1 = weiß/orange
* Pin 2 = orange
* Pin 3 = weiß/grün
* Pin 4 = blau
* Pin 5 = weiß/blau
* Pin 6 = grün
* Pin 7 = weiß/braun
* Pin 8 = braun

### Messungen bei vollständig ausgeschalteter Wärmepumpe

Widerstandsmessung direkt über die RJ45-Buchse:

* Pin 1 ↔ 2: **ca. 1,2 Ω**
* Pin 3 ↔ 6: **ca. 1,1 Ω**
* Pin 4 ↔ 5: **ca. 0,4 Ω**
* Pin 7 ↔ 8: **ca. 0,4 Ω**

Zwischen den daraus resultierenden vier Gruppen messe ich dagegen ziemlich reproduzierbar ungefähr **150 Ω**:

* Pin 3 ↔ 4: **149,8 Ω**
* Pin 4 ↔ 6: **149,8 Ω**
* Pin 1 ↔ 3: **150,4 Ω**
* Pin 1 ↔ 4: **149,6 Ω**
* Pin 1 ↔ 7: **150,8 Ω**
* Pin 3 ↔ 7: **ca. 151 Ω**
* Pin 4 ↔ 7: **150,1 Ω**

Für mich sieht das so aus, als ob die RJ45-Kontakte vier Gruppen bilden:

**1/2 – 3/6 – 4/5 – 7/8**

Interessant ist, dass ich damit zwischen den möglichen Signalgruppen nicht die erwarteten ~60 Ω für einen mit 2×120 Ω terminierten CAN-Bus messe, sondern jeweils ungefähr 150 Ω.

### Messungen bei eingeschalteter Wärmepumpe

Anschließend habe ich die Anlage wieder vollständig eingeschaltet und mit dem Fluke im DC-V-Bereich gemessen.

Zwischen:

* 1 ↔ 3
* 1 ↔ 4
* 1 ↔ 7
* 3 ↔ 4
* 3 ↔ 7
* 4 ↔ 7

messe ich jeweils **0,000 V DC**.

Das Multimeter bzw. die Messleitungen habe ich gegengeprüft: Eine Batterie wird korrekt mit ca. **1,4 V DC** gemessen.

Damit finde ich an der RJ45-Buchse weder die erwarteten **19–30 V zwischen VCC/GND** noch die erwarteten ungefähr **2,5 V CAN-Pegel gegen GND**.

Auf dem Schaltplan meiner Anlage ist die Buchse allerdings eindeutig als **„RJ45 CAN“** bezeichnet und als interne Verbindung des RoCon BM2C zum RoCon+ Bedienteil dargestellt. J13 ist separat mit **CAN-H / CAN-L / CAN-GND / CAN-VCC** beschriftet.

Mein geplanter Aufbau wäre:

**RJ45 der Altherma → M5Stack ATOMIC CAN Base (CA-IS3050G) → AtomS3 Lite → ESPHome**

Den AtomS3 würde ich zunächst separat über USB-C versorgen und CAN-VCC keinesfalls verwenden.

Bevor ich irgendetwas mit dem M5Stack verbinde, hätte ich daher ein paar Fragen an diejenigen, die den RoCon BM2C bzw. genau diese RJ45-Buchse kennen:

1. **Kennt jemand die verifizierte Pinbelegung dieser RJ45-CAN-Buchse am RoCon BM2C?**
2. Sind die Gruppen **1/2, 3/6, 4/5 und 7/8** tatsächlich jeweils parallel geschaltete Kontakte oder interpretiere ich meine Widerstandsmessung falsch?
3. Warum messe ich zwischen allen vier Gruppen ungefähr **150 Ω**, aber bei eingeschalteter Anlage **0 V DC**?
4. Ist diese RJ45-Buchse tatsächlich direkt als CAN-Anschluss für einen externen CAN-Transceiver nutzbar oder sitzt zwischen RJ45 und dem eigentlichen J13-CAN noch Elektronik?
5. Hat jemand einen **M5Stack AtomS3 Lite + ATOMIC CAN Base** bereits erfolgreich **direkt an dieser RJ45-Buchse** betrieben?
6. Falls ja: Welche RJ45-Pins gehen auf **CAN-H, CAN-L und CAN-GND**?
7. Wird bei diesem Anschluss ein zusätzlicher **120-Ω-Abschlusswiderstand** benötigt oder sollte der ATOMIC CAN Base ohne zusätzlichen Terminator angeschlossen werden?
8. Reicht es, **CAN-H, CAN-L und CAN-GND** anzuschließen und den AtomS3 separat per USB zu versorgen, sodass CAN-VCC vollständig unbenutzt bleibt?
9. Gibt es eine Möglichkeit, die korrekte Belegung noch **nichtinvasiv an der RJ45-Buchse** zu verifizieren? Direkt an J13 möchte ich aufgrund der Mietanlage nicht messen.

Mir geht es vor allem darum, nichts durch Trial-and-Error an der Heizungssteuerung zu beschädigen. Falls jemand einen Schaltplan bzw. ein Pinout genau dieser RJ45-CAN-Verbindung RoCon BM2C ↔ RoCon+ B1 hat, wäre das extrem hilfreich.

Vielen Dank!
NetRat88

**Comment by wrfz** [comment]
**Zu den Pin-Paaren:**
Es handelt sich um die Paare DA (1/2), DB (3/6), DC (4/5) und DD (7/8) gemäß TIA-568B (siehe die große Tabelle unter https://de.wikipedia.org/wiki/TIA-568A/B).

Ein Standard-RJ45-Kabel mit T-568B-Belegung kann problemlos für das CAN-Protokoll genutzt werden.
CAN benötigt an den beiden physisch äußersten Enden des Busses jeweils einen Abschlusswiderstand von 120 Ω. Dadurch ergibt sich im laufenden Bus ein Gesamtwiderstand von ca. 60 Ω zwischen CAN_H und CAN_L. Bei kurzen Leitungslängen funktioniert die Kommunikation oft auch ohne diesen Abschluss. Dass du keine 60 Ω misst, liegt sehr wahrscheinlich daran, dass die Bus-Terminierung auf der Platine nicht vorhanden ist.

**Zu den 150 Ω und den 0 V DC**
Ein gewöhnliches Multimeter ist für schnelle Digitalsignale zu langsam. Um den Pegelwechsel auf den CAN-Leitungen darzustellen, benötigst du ein Oszilloskop.

Die 0 V DC erklären sich wie folgt: Ein CAN-Bus befindet sich im ruhenden Zustand (rezessiv) typischerweise auf einem Differenzpegel von 0 V (beide Leitungen liegen auf ca. 2,5 V gegen Masse). Wenn Daten übertragen werden (dominant), zieht CAN_H auf ca. 3,5 V an und CAN_L fällt auf ca. 1,5 V ab. Die Differenz beträgt dann ca. 2 V. Da ein Multimeter nur den zeitlichen Gleichspannungs-Mittelwert anzeigt, misst man im Leerlauf oder bei kurzer Aktivität nahe 0 V Differenzspannung.

Der gemessene Wert von 150 Ω spricht dafür, dass an einem Ende ein interner Abschlusswiderstand anliegt (oft 120 Ω bis 150 Ω), der Bus aber nicht vollständig an beiden Enden korrekt terminiert ist.

**Zum M5Stack AtomS3 Lite + ATOMIC CAN Base**
Das hängt nicht von der RJ45-Buchse ab, sondern von der Signalbelegung der Anlagenelektronik: Befinden sich auf diesem Port die vollständigen Steuer- und Sensordaten oder nur ein eingeschränktes Protokoll für das Display? In der Community ist dazu bisher kein erfolgreicher Einsatz bekannt, technisch ist es bei passender Pegelanpassung und Baudrate jedoch denkbar.

**Comment by NetRat88** [comment]
Danke für die Rückmeldung! Was mich noch stutzig macht: Ich messe die ~150 Ω praktisch zwischen allen vier Gruppen DA, DB, DC und DD. Bei eingeschalteter Anlage messe ich zwischen allen vier Gruppen untereinander jeweils 0,000 V DC.

Da wir mit dem Multimeter hier offenbar nicht weiterkommen: Hat jemand ein Oszilloskop bzw. geeignetes CAN-Messequipment und eine Altherma mit RoCon BM2C und könnte die RJ45-CAN-Buchse einmal messen?

Interessant wäre vor allem, auf welchen Pins tatsächlich CAN-Kommunikation stattfindet, welche Pegel und Baudrate anliegen und ob sich CAN-H/CAN-L eindeutig identifizieren lassen.

Dann könnten wir klären, ob sich diese RJ45-Buchse als nichtinvasiver Anschluss für das Projekt nutzen lässt, ohne direkt an J13 gehen zu müssen.

Mir fehlen dafür leider die entsprechenden Messmittel und auch die Erfahrung mit CAN-Signalanalyse. Vielleicht kennt alternativ jemand Serviceunterlagen, Schaltpläne oder eine andere Quelle, aus der sich die elektrische Belegung bzw. das Protokoll dieser RJ45-CAN-Verbindung entnehmen lässt?

**Comment by KalaNagHTD** [comment]
Auch meine „Ultra“ hat an der von außen zugänglichen RJ45-Buchse des Bedienteils keinerlei Spannung oder Signale!
